### PR TITLE
support reordering packages in system site

### PR DESCRIPTION
**Pull request by karlicoss**
should resolve https://github.com/seanbreckenridge/reorder_editable/issues/3

### CODE REVIEW

File: reorder_editable/__main__.py [hunk 1]
```diff
@@ -52,30 +52,42 @@ def _print_editable_contents(
...
         editable_pth = chosen_editable
     else:
-        editable_pth = _resolve_editable()
+        editable_pth = _resolve_editable(use_user_site=use_user_site)
     with open(editable_pth, "r") as src:
         click.echo(src.read(), nl=False, err=stderr)
 
 
+site_option = click.option(
+    '--user/--system',
```
[the remark this answers is not in the document]
Comment (seanbreckenridge) [reply reorder_editable/__main__.py:61]: yeah, this is the correct way to do this :+1:

**Review by seanbreckenridge — APPROVED** [review]
yeah, I think its fine to pass `use_user_site` everywhere here, I expect this'll change again with the new PEP stuff

**Comment by seanbreckenridge** [comment]
Thanks 👍

Will make a release later today

**Comment by karlicoss** [comment]
Btw, I think because the CI is only configured to run on master, I couldn't run Actions against my fork's CI. Perhaps worth making it run on push to any branch, e.g. like here: https://github.com/karlicoss/pymplate/blob/9e79bc101b8abe2fa1fde3c09d7b2345b2164b59/.github/workflows/main.yml#L6

**Comment by seanbreckenridge** [comment]
Ah yeah, will write a script to do that for me

**Comment by seanbreckenridge** [comment]
Ok nevermind, was such a mess dealing with the yaml file and trying to keep the same ordering, I just did a quickfix list (similar to compilation mode I think)

`nvim -q +copen <(fd --full-path -H --glob '**/workflows/*.y*ml' -X rg --no-heading --with-filename --line-number 'branches.*(main|master)')`

And then just had a macro that edited the file, added it to git and pushed

`"q   gg/branches^Mf:<80><fd>alld$A['*']^[nf:<80><fd>ald$a ['*']^[:w^M:Git add % | Git commit -m 'run a<80>kbCI on all branches' | Git push`

took me like 5 minutes instead of the 30 it did to try and write the script...

**Pull request merged**